MIT Press.js

```diff
@@ -0,0 +1,206 @@
+{
+	"translatorID": "ac277fbe-000c-46da-b145-fbe799d17eda",
+	"label": "MIT Press",
```

Comment:
Should be called "MIT Press Books" to distinguish it from existing "MIT Press Journals" translator